runtime/queries/rust/highlights.scm

```diff
@@ -103,72 +50,173 @@
   "<" @punctuation.bracket
   ">" @punctuation.bracket)
 
-"::" @punctuation.delimiter
-"." @punctuation.delimiter
-";" @punctuation.delimiter
+(parameter
+	pattern: (identifier) @variable.parameter)
+(closure_parameters
+	(identifier) @variable.parameter)
 
-(parameter (identifier) @variable.parameter)
-(closure_parameters (_) @variable.parameter)
 
-(lifetime (identifier) @label)
+; Other Global
+
+
+(impl_item
+  "for" @keyword)
+
+"loop" @special
+"for" @special
+"in" @special
+"break" @special
+"continue" @special
+"while" @special
+
+"match" @special
+"if" @special
+"else" @special
+"await" @special
+"return" @special
 
-"async" @keyword
-"break" @keyword
-"const" @keyword
-"continue" @keyword
 (crate) @keyword
-"default" @keyword
-"dyn" @keyword
-"else" @keyword
-"enum" @keyword
 "extern" @keyword
```

Comment:
Can we just put all these in an array? `[....] @keyword`. Each item needs to have either 0 or 1 space of indentation, otherwise it seems to skip the definition.